source/addonAPIVersion.py

```diff
@@ -13,7 +13,7 @@
 """
 
 CURRENT = (buildVersion.version_year, buildVersion.version_major, buildVersion.version_minor)
-BACK_COMPAT_TO = (2019, 3, 0)
+BACK_COMPAT_TO = (2021, 1, 0)
 """
 As BACK_COMPAT_TO is incremented, the changed / removed parts / or reasoning should be added below.
 EG: (x, y, z): Large changes to speech.py

```

Comment:
I think there should be some reasoning below this line about the version updating.

Reply:
Hmm, yes you are right. Though given the size of the list, it might be better to refer to the changelog.